src/pages/Stream/Views/Explore/StaticLogTable.tsx

```diff
@@ -240,7 +249,7 @@ const Footer = (props: { loaded: boolean }) => {
 	return (
 		<Stack className={tableStyles.footerContainer} gap={0}>
 			<Stack w="100%" justify="center" align="flex-start">
-				{!props.loaded ? null : <TotalLogsCount />}
+				{!props.loaded ? <Loader type="dots" /> : <TotalLogsCount />}
```

Comment:
This still takes the loading state of the table data. You should implement a separate loading state on fetchQueryCount.